### PR TITLE
Add Spanish translation of Advanced User - graphQL

### PR DESCRIPTION
**Pull request by bezaca**
Hi there :handshake:!
This pull request include the Spanish translation of advanced/graphql.md 

This pull request continues with the Spanish translation of #1218

### CODE REVIEW

File: docs/es/docs/advanced/graphql.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# GraphQL
+
+**FastAPI** tiene soporte opcional para GraphQL (proveído por Starlette directamente), usando la librería de `graphene`.
```
Comment (anthonycepeda) [inline docs/es/docs/advanced/graphql.md:3]: Hi, maybe: 
**FastAPI** cuenta con soporte opcional para GraphQL, usando la librería `graphene` de Startlette.

File: docs/es/docs/advanced/graphql.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Puedes combinar *operaciones de <abbr title="ruta">path</abbr>* normales de FastAPI con GraphQL en la misma aplicación.
+
+## Importar y utilizar `graphene`
+
+GraphQL es implementado mediante Graphene, puedes mirar la <a href="https://docs.graphene-python.org/en/latest/quickstart/" class="external-link" target="_blank"> documentación de Graphene </a> para más detalles.
```
Comment (anthonycepeda) [inline docs/es/docs/advanced/graphql.md:9]: GraphQL ha sido implementado usando Graphene, puedes revisar la documentación oficial de Graphene para más detalles

File: docs/es/docs/advanced/graphql.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Ejecutalo con Uvicorn y abre tu navegador en <a href="http://127.0.0.1:8000" class="external-link" target="_blank">http://127.0.0.1:8000</a>.
+
+Verás la interfaz web de usuarios de GraphQL :
+
+<img src="https://fastapi.tiangolo.com/img/tutorial/graphql/image01.png">
+
+## Más detalles
+
+Para más detalles , incluyendo :
```
Comment (anthonycepeda) [inline docs/es/docs/advanced/graphql.md:38]: Para conocer más detalles sobre:

**Review by anthonycepeda — CHANGES_REQUESTED** [review]
Hi,
I've added some changes request.
Thanks

**Comment by tiangolo** [comment]
Thanks @bezaca! 🤓 

As this section changed a lot in the original in English, I'm gonna pass on this one, but thanks for the effort! ☕

**Pull request closed without merging**